core/quantity/src/Parser.ts

```diff
@@ -333,22 +333,33 @@ export class Parser {
           }
         } else {
           // not processing a number
-          if (isOperator(charCode)) {
+          const isCharOperator = isOperator(charCode);
+          const isSpacer = charCode === format.spacerOrDefault.charCodeAt(0) && charCode !== QuantityConstants.CHAR_SPACE;
```

Comment:
Just to check, are you intentionally disallowing space even if `spacer` is set to space? Because otherwise, I think the following would be better:

```suggestion
          const isSpacer = charCode === format.spacer.charCodeAt(0);
```

Note: `spacerOrDefault` returns `spacer` if it has a value, or `" "` if it is undefined. But if `spacer` is **set** to space, it seems like it should be allowed.

Reply:
Never mind. Even if `spacer` is set to space, you don't want `isSpacer` to be true.